query.go

```diff
@@ -96,6 +98,9 @@ func (q *Query) StopTime() time.Time {
 // Set the start time of the query
 func (q *Query) SetStartTime(s time.Time) {
 	q.startTL.Val = s
+	if q.alignGroup && q.groupByOffsetDL != nil {
+		q.groupByOffsetDL.Val = s.Sub(time.Unix(0, 0))
```

Comment:
✔️

Thanks for doing the arithmetic on that, I wasn't confident it was safe to do even though it seemed reasonable.  [-x,0] is covered by [0,x], so I implemented it as such.